examples/scaleway/main.tf

```diff
@@ -60,131 +48,290 @@ module "ingress" {
 }
 
 module "cert-manager" {
-  source = "git::https://github.com/camptocamp/devops-stack-module-cert-manager.git//scaleway?ref=remove-read-only-attribut"
+  source = "git::https://github.com/camptocamp/devops-stack-module-cert-manager.git//self-signed?ref=v8.1.0"
 
-  cluster_name     = local.cluster_name
+  enable_service_monitor = var.cert_manager_enable_service_monitor
+
+  dependency_ids = {
+    argocd = module.argocd_bootstrap.id
+  }
+}
+
+module "authentication_with_keycloak" {
```

Comment:
On the other examples we called this `keycloak`, I think we should carry the name for consistency.